src/migration.types.ts

```diff
@@ -171,3 +171,11 @@ export type MigrationFunction<
     doesReportIncludeStacks: boolean;
   },
 ) => void;
+
+/**
+ * The behavior to have when an error occurs during the migration of an item:
+ * - "keep-original": if anything goes wrong during the migration of an item, the CLI `from-version` of the item is kept.
+ * - "keep-latter": if anything goes wrong during the migration of an item between n-version and n+1-version, the n-version of the item is kept.
+ * - "keep-going":  if anything goes wrong during the migration of an item, keep trying to migrate it, even though it's corrupt, until the CLI `to-version`.
```

Comment:
```suggestion
 * - "keep-going":  the n-p remaining steps are applied on the version of the item obtained after the first p-1 successful steps.
```

We could also go for "skip". What do you think ?

Reply:
I find `"keep-going"` clearer and actually more accurate. With `"skip"`, I feel like people could understand "skip the migration", as in "don't migrate this item at all", instead of "skip the error". What do you think?

Reply:
`"keep-going"` sounds good then